src/main/javatest/com/google/security/cryptauth/lib/securegcm/Ukey2ShellCppWrapper.java

```diff
@@ -50,6 +51,7 @@
 public class Ukey2ShellCppWrapper {
   // The path the the ukey2_shell binary.
   private static final String BINARY_PATH = "build/src/main/cpp/src/securegcm/ukey2_shell";
```

Comment:
we don't build w cmake anymore, so this should probably be removed.

Reply:
Are you planning to support only bazel for the c++ library and gradle for java?

Reply:
yep, since this is built as part of https://github.com/google/nearby, so that's why a lot of this stuff wasn't updated correctly (my bad!)